src/libs/ajax-fetch.js

```diff
@@ -45,3 +45,9 @@ export const fetchCbas = withUrlPrefix(`${getConfig().cbasUrlRoot}/api/batch/v1/
 export const fetchCromwell = withUrlPrefix(`${getConfig().cromwellUrlRoot}/api/workflows/v1/`, fetchOk)
 export const fetchLeo = withUrlPrefix(`${getConfig().leoUrlRoot}/`, fetchOk) // TODO: How to add this config to Cromwhelm?
 export const fetchWds = wdsUrlRoot => withUrlPrefix(`${wdsUrlRoot}/`, fetchOk)
+
+const wsmRoot = getConfig().workspaceManagerRoot ?
+  `${getConfig().workspaceManagerRoot}/api/workspaces/v1` :
+  `https://workspace.dsde-dev.broadinstitute.org/api/workspaces/v1`
```

Comment:
Curious about the dev fallback value here. I don't think you should worry about providing a url value if the environment config fails to do so, but I could be missing some context here.